### PR TITLE
WIP: Moved some folders around

### LINKED ISSUE #1251: Move some folders around
Only for `packages/docs`.

* move `iconset` to `components`. There is too few icons to justify a separate folder.
* move `utilities/markdown-view` to `components`
* move `utils` to `helpers/` and name in a way that makes sense, or include in existing helpers file.

Doesn't have to be strictly as I wrote, but we want to move these out of root `src/`

### PR DESCRIPTION
**Pull request by ishpartko**
## Description

closes #1251

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/helpers/ReadHelper.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+export class ReadHelper {
```
Comment (asvae) [inline packages/docs/src/helpers/ReadHelper.ts:2]: Components could be functions, no point to wrap into class.

File: packages/docs/src/pages/MarkdownTest.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <script lang="ts">
 // @ts-ignore
 import { Options, Vue } from 'vue-class-component'
-import MdView from '../utilities/markdown-view/MarkdownView.vue'
+import MdView from '../components/MarkdownView/MarkdownView.vue'
```
Comment (asvae) [inline packages/docs/src/pages/MarkdownTest.vue:8]: Could use full name - MarkdownView. (nitpicking :))

**Comment by asvae** [comment]
Looks good 👍 . Please merge after fixes.

**Review by asvae — APPROVED** [review]

**Comment by ishpartko** [comment]
depends on https://github.com/epicmaxco/vuestic-ui/pull/1280

**Pull request merged**